### PR TITLE
Update accessibility statement re last TDT issue

### PR DESCRIPTION
**Pull request by EoinShaughnessy**
Fixes [#1792](https://github.com/alphagov/govuk-design-system/issues/1792).

Following the [release of the latest version of the Tech Docs Template](https://github.com/alphagov/tech-docs-gem/blob/master/CHANGELOG.md#240), there is only one remaining issue making our Frontend docs site non-compliant with WCAG:

- When a user searches, the search results dialogue hides the main page region from assistive technology. This fails [WCAG 2.1 success criterion 2.4.3 Focus Order.](https://www.w3.org/TR/WCAG21/#focus-order) and [WCAG 2.1 success criterion 3.2.2 - On Input](https://www.w3.org/TR/WCAG21/#on-input).

This PR updates our [accessibility statement](https://design-system.service.gov.uk/accessibility/#non-accessible-content) to say:

- there is a non-compliant issue (instead of issues)
- when we plan to fix the outstanding issue

### CODE REVIEW

File: src/accessibility.md.njk
Comment (EoinShaughnessy) [inline src/accessibility.md.njk:61]: This text will serve as a placeholder we can iterate on.

Are we required to say _when_ we intend to fix the issue by?

Comment (EoinShaughnessy) [reply src/accessibility.md.njk:61]:
```suggestion
We plan to fix this issue by 30 September 2021
```
Basing this ^ on line 70 here: https://github.com/alphagov/tdt-documentation/pull/163/files

Comment (EoinShaughnessy) [reply src/accessibility.md.njk:61]: Ok, we're not to include a date for fixes.

**Review by lfdebrux — APPROVED** [review]
Content change looks good to me 👍 

I'm guessing we want to wait until the changes to tech-docs-template have been merged in https://github.com/alphagov/govuk-frontend-docs/pull/139 before merging this?

**Comment by EoinShaughnessy** [comment]
Thanks, @lfdebrux! Yeah, once we merge [#139](https://github.com/alphagov/govuk-frontend-docs/pull/139), I'll merge this PR.

**Comment by 36degrees** [comment]
@EoinShaughnessy https://github.com/alphagov/govuk-frontend-docs/pull/139 has now been merged 👍🏻

**Pull request merged**
